handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -121,67 +114,40 @@ public void handle(Invocation invocation, AsyncResponse asyncResp) throws Except
   }
 
   private void clearLoadBalancer() {
-    for (LoadBalancer loadBalancer : loadBalancerMap.values()) {
-      loadBalancer.shutdown();
+    for (LoadBalancerCreator creator : loadBalancerMap.values()) {
+      creator.shutdown();
     }
     loadBalancerMap.clear();
   }
 
-  protected void setIsolationFilter(LoadBalancer lb, String microserviceName) {
-    final String filterName = IsolationServerListFilter.class.getName();
-    IsolationServerListFilter isolationListFilter = new IsolationServerListFilter();
-    isolationListFilter.setMicroserviceName(microserviceName);
-    isolationListFilter.setLoadBalancerStats(lb.getLoadBalancerStats());
-    lb.putFilter(filterName, isolationListFilter);
-  }
-
-  protected void setTransactionControlFilter(LoadBalancer lb, String microserviceName) {
-    final String filterName = TransactionControlFilter.class.getName();
+  protected void setTransactionControlFilter(String microserviceName) {
     String policyClsName = Configuration.INSTANCE.getFlowsplitFilterPolicy(microserviceName);
-    if (policyClsName.isEmpty()) {
-      return;
-    }
-    try {
-      Class<?> policyCls = Class.forName(policyClsName);
-      if (!TransactionControlFilter.class.isAssignableFrom(policyCls)) {
-        String errMsg = String.format(
-            "Define instance filter %s in yaml, but not extends abstract class TransactionControlFilter.",
-            policyClsName);
-        LOGGER.error(errMsg);
-        throw new Error(errMsg);
-      }
-      TransactionControlFilter transactionControlFilter = (TransactionControlFilter) policyCls.newInstance();
-      transactionControlFilter.setLoadBalancerStats(lb.getLoadBalancerStats());
-      transactionControlFilter.setMicroserviceName(microserviceName);
-      lb.putFilter(filterName, transactionControlFilter);
-    } catch (Throwable e) {
-      String errMsg = "Fail to create instance of class: " + policyClsName;
-      LOGGER.error(errMsg);
-      throw new Error(errMsg, e);
+    if (!policyClsName.isEmpty()) {
+      LOGGER.error(Configuration.TRANSACTIONCONTROL_POLICY_KEY_PATTERN + " is not supported anymore." +
+          "You can change this class to SPI, and filters will be loaded by SPI.");
```

Comment:
We need to provide a document for the user to upgrade the code the new SPI.

Reply:
Document added : https://huaweicse.github.io/servicecomb-java-chassis-doc/zh_CN/references-handlers/loadbalance.html